Header header
geometry_msgs/PoseWithCovariance[] gt
geometry_msgs/PoseWithCovariance[] rad
geometry_msgs/PoseWithCovariance[] leg
geometry_msgs/PoseWithCovariance[] deep